#file_name: /home/amov/IPN_MPC/data/traj/Traj.txt
#file_name: /home/amov/IPN_MPC/data/traj/Traj_gerono_radius_15_vs_2.txt
#file_name: /home/amov/IPN_MPC/data/traj/Traj_wobbly_radius1.5_vs_4.txt
#file_name: /home/amov/IPN_MPC/data/traj/Traj_vert_osc.txt

file_name: /home/amov/IPN_MPC/data/traj/Traj_for_dyn.txt

control_freq: 100
